### ISSUE TITLE
Upgrade libraries to make NextGen Connect compatible with latest Apache-POI

### ISSUE THREAD

**Issue by is-simon** [nextgenhealthcare/connect#4897]
**Is your feature request related to a problem? Please describe.**

We have an home made extension that uses Apache-POI and we are trying to upgrade our Apache-POI dependencies to the latest one (currently 5.1.0) and came accross conflicted dependencies with the NextGen connect ones. We would like to suggest an upgrade of 2 libraries that could make NextGen Connect compatible with the latest Apache-POI version. 

**Describe your use case**
We need to upgrade our Apache-POI (3.17) dependencies to the latest one (5.1.0), and cannot because of 2 NextGen Connect libraries, **commons-io-2.6.jar** (server lib) and **xbean-fixed-2.4.0.jar** (ws extension lib)

**Describe the solution you'd like**
The latest version of Apache-POI requires **commons-io-2.7.jar** and later (compile dependency is 2.11) , and **xbean-3.1.0.jar** and later (compile dependency is 5.0.2) .

**Comment by tonygermano** [comment]
This can likely be resolved by using an isolatedClassLoader

**Comment by is-simon** [comment]
> This can likely be resolved by using an isolatedClassLoader

I don't think so (but maybe I'm wrong). Our extension is a "`ServicePlugin`" extension and the libraries appear in the plugin.xml file of the extension:

```
<library type="SERVER" path="lib/curvesapi-1.04.jar" />
<library type="SERVER" path="lib/poi-3.17.jar" />
<library type="SERVER" path="lib/poi-ooxml-3.17.jar" />
<library type="SERVER" path="lib/poi-ooxml-schemas-3.17.jar" />
<library type="SERVER" path="lib/poi-scratchpad-3.17.jar" />
```

So they are loaded by the extensions initialization mechanism, not by us.

Also, the extension offers a userutil package with functions that can be called from a Filter/Transformer in a channel.

**Comment by tonygermano** [comment]
You are correct. I overlooked the part that said it was an extension. The request is somewhat related to this issue I opened a couple years ago. https://github.com/nextgenhealthcare/connect/issues/4350.
